vitrina/smart_contracts/migrations/0005_create_initial_default_smart_contract_template.py

```diff
@@ -0,0 +1,99 @@
+# Generated by Django 3.2.25 on 2025-07-25 12:22
+from django.core.files.base import ContentFile
+from django.db import migrations
+
+default_smart_contract_template = """
+### **DUOMENŲ TEIKIMO-GAVIMO SUTARTIS**
+
+{{ odrl_data["issued"] }}, Vilnius
+
+{{ odrl_data["assigner"][0]["ex:companyName"] }}, juridinio asmens kodas {{ odrl_data["assigner"][0]["ex:companyCode"] }}, registruotos buveinės adresas {{ odrl_data["assigner"][0]["ex:address"] }}, atstovaujama/as {{ odrl_data["assigner"][0]["ex:representative"] }}, veikiančio pagal Teikėjas įgaliojimus (toliau – **Duomenų teikėjas**)
+
+ir
+
+{{ odrl_data["assignee"][0]["ex:companyName"] }}, juridinio asmens kodas {{ odrl_data["assignee"][0]["ex:companyCode"] }}, registruotos buveinės adresas {{ odrl_data["assignee"][0]["ex:address"] }}, atstovaujama/as {{ odrl_data["assignee"][0]["ex:representative"] }}, veikiančio pagal Gavėjo įgaliojimus (toliau – **Duomenų gavėjas**),
+
+toliau Duomenų teikėjas ir Duomenų gavėjas kartu vadinami Šalimis, o kiekvienas atskirai – Šalimi, sudaro šią duomenų teikimo-gavimo sutartį (toliau – Sutartis):
+
+**1 SĄVOKOS**
+
+1.1 Šioje Sutartyje toliau nurodytos sąvokos, parašytos iš didžiosios raidės, turi tokias reikšmes:
+
+1.1.1 **Duomenys** – aktų, faktų ar informacijos ir tokių aktų, faktų ar informacijos rinkinių skaitmeninė Duomenys – aktų, faktų ar informacijos ir tokių aktų, faktų ar informacijos rinkinių skaitmeninė Duomenys – aktų, faktų ar informacijos ir tokių aktų, faktų ar informacijos rinkinių skaitmeninė
```

Comment:
This sentence is copied & pasted multiple times? Does not make sense.

Also, "skaitmeninė" what? Missing the end for that sentence

Reply:
🦅

Reply:
Looks like leftover from previous task, great job!